common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -194,26 +198,23 @@ def test_import_timestamp(self):
             And if I refresh the page, the timestamp is still displayed
         """
         self.assertFalse(self.import_page.is_timestamp_visible())
+
+        # Get the time when the import has started.
+        upload_start_time = datetime.utcnow().replace(microsecond=0, second=0)
```

Comment:
Adam the timestamp shown on the page is in the form of date time. `(MM/DD/YYYY at HH:mm)` and the current time is in the form of `(MM/DD/YYYY at HH:mm ss mc)`. Means it include `microsecond` and `seconds` which cause inconsistency in comparing.

So by replacing we are comparing `(MM/DD/YYYY at HH:mm)`  with `(MM/DD/YYYY at HH:mm)`